src/lib.rs

```diff
@@ -21,14 +21,15 @@ pub mod crossover;
 pub mod error;
 /// Fee
 pub mod fee;
-/// Message representation
-pub mod message;
 /// Transparent and Obfuscated Notes
 pub mod note;
 
 /// Phoenix Core Keys & Addresses
 mod keys;
 
+/// Encryption and decryption methods
+pub mod encryption;
```

Comment:
```suggestion
mod encryption;
```
I would suggest to not make this a public module but instead re-export the two functions `encrypt` and `decrypt` in lib.rs:
```
pub use encryption::{encrypt, decrypt};
```
This way we avoid the `encryption::encrypt` path and it's generally easier to see which types are exposed and which are not when the modules are private in general.